.github/workflows/docker_build.yml

```diff
@@ -1,18 +1,58 @@
-name: Publish Docker
-on: 
+# -------------------------------------------------------------------------------------------------
+# Job Name
+# -------------------------------------------------------------------------------------------------
+name: Build-Publish
+
+# -------------------------------------------------------------------------------------------------
+# When to run
+# -------------------------------------------------------------------------------------------------
+on:
   push: {}
   schedule:
     - cron:  '0 0 * * *'
+
+# -------------------------------------------------------------------------------------------------
+# What to run
+# -------------------------------------------------------------------------------------------------
 jobs:
-  build-publish:
+  build:
+    name:  "[ ${{ matrix.TF_VERSION }}-${{ matrix.TG_VERSION }} ]"
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: False
+      matrix:
+        TF_VERSION: [0.12.29,latest]
```

Comment:
Let's use `0.12` (major/minor) version only and have the Dockerfile figure out the latest patch level version.

Reply:
This has been already added, for TG still added an specific patch level which is still needed